app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,40 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        =bootstrap_form_for [:admin, @video], html: {class: "form-horizontal"} do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="/ui/admin_views_payments") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset
+            .form-group
+              .control-label.col-sm-3= f.label :title, 'Title'
+              .col-sm-3
+                = f.text_field :title, hide_label: true
+            .form-group
+              .control-label.col-sm-3= f.label :category_id, 'Category'
+              .col-sm-3
+                = f.select :category_id, options_for_select(Category.all.map{|category| [category.title, category.id]}), hide_label: true
+            .form-group
+              .control-label.col-sm-3= f.label :description, 'Description'              
+              .col-sm-6
+                = f.text_field :description, hide_label: true
```

Comment:
HOMEWORK: rework this whole form after reading docs more and refactor it to nicely looking nifty code.